tests/testthat/test-add_annotation.R

```diff
@@ -13,81 +11,82 @@
 #' T3.2 The content of a `data.frame` passed to label is not affected by the transformation to an annotation
 #' T3.3 The content of a `gtable` passed to label is not affected by the transformation to an annotation
-#' T4. The annotation can be placed on the plot by specifying the coordinates 
+#' T4. The annotation can be placed on the plot by specifying the coordinates
 #' T4.1 An error when one of the coordinates is not numeric
 #' T4.2 The annotation can be moved on the plot by specifying the x coordinates
 #' T4.3 The annotation can be moved on the plot by specifying the y coordinates
 #' T5. The layout of the annotation can be modified to a certain extend
 #' T5.1 The annotation has bold columnheaders when the passed object is of class `data.frame`
 #' T5.2 The font size can be changed
-#' T5.3 The font family can be chosen between "sans", "serif" and "mono"
+#' T5.3 The font family can be chosen between 'sans', 'serif' and 'mono'
 #' T6. The output object has an additional attribute `components`
 #' T6.1 The attribute components[['visR_plot']] contains the plot used as input
 #' T6.2 The attribute components contains the annotation
 #' T6.3 The output has the same class as the original ggplot
 
+# Requirement T1 ---------------------------------------------------------------
 
-# Requirement T1 ----------------------------------------------------------
-
-context("add_annotation - T1. The function adds annotations to an object of class `ggplot`")
+testthat::context("add_annotation - T1. The function adds annotations to an object of class `ggplot`")
 
-testthat::test_that("T1.1 No error when a `ggplot` object is passed to the function in the presence of a label",{
+testthat::test_that("T1.1 No error when a `ggplot` object is passed to the function in the presence of a label", {
 
-  visR_plot <- visR::estimate_KM(data = adtte, strata = "TRTA") %>% visR::visr()
+  visR_plot <- adtte %>%
+    visR::estimate_KM(strata = "TRTA") %>% 
+    visR::visr()
 
-  testthat::expect_error(visR::add_annotation(visR_plot, label = "blah"), NA)
+  testthat::expect_error(visR_plot %>% visR::add_annotation(label = "blah"), NA)
 
 })
 
-testthat::test_that("T1.2 An error when a non-`ggplot` object is passed to the function in the presence of a label",{
+testthat::test_that("T1.2 An error when a non-`ggplot` object is passed to the function in the presence of a label", {
   
   visR_plot <- "blah"
 
   testthat::expect_error(visR::add_annotation(gg = visR_plot, label = "blah"))
 
 })
 
-testthat::test_that("T1.3 An error when NULL is passed to the function in the presence of a label",{
+testthat::test_that("T1.3 An error when NULL is passed to the function in the presence of a label", {
 
   testthat::expect_error(visR::add_annotation(gg = NULL, label = "blah"))
 
 })
 
 
-# Requirement T2 ----------------------------------------------------------
+# Requirement T2 ---------------------------------------------------------------
```

Comment:
The length of '-----" should be equal to the one of the insert-> section of R studio - length of text preceding it

Reply:
It is now 80 char, some of them were just shorter, probably because they were manually written

Reply:
Okay, RStudio seems to leave an arbitrary print margin? For me tests files are not included in the pdf, can you check if 80 also works? 75 feels arbitrary